manifests/nexus.pp

```diff
@@ -44,6 +44,7 @@
   Optional[String]  $proxy_server    = undef,
   Optional[String]  $proxy_type      = undef,
   Optional[Boolean] $allow_insecure  = undef,
+  Optional[String]  $temp_dir        = undef,
```

Comment:
I'd have probably gone for `Optional[Stdlib::Absolutepath]`

Reply:
As you wish :)